services/web/server/tests/unit/with_dbs/test_access_to_studies.py

```diff
@@ -154,12 +155,43 @@ async def _get_user_projects(client):
 def _assert_same_projects(got: Dict, expected: Dict):
     # TODO: validate using api/specs/webserver/v0/components/schemas/project-v0.0.1.json
     # TODO: validate workbench!
-    exclude = ["creationDate", "lastChangeDate", "prjOwner", "uuid", "workbench", "accessRights"]
+    exclude = [
+        "creationDate",
+        "lastChangeDate",
+        "prjOwner",
+        "uuid",
+        "workbench",
+        "accessRights",
+    ]
```

Comment:
```suggestion
    exclude = set(
        "creationDate",
        "lastChangeDate",
        "prjOwner",
        "uuid",
        "workbench",
        "accessRights",
    )
```
in operator is faster with sets with respect to lists

Reply:
thx. it won't make a big difference here, but it is good to know :-)